packages/renderer/src/lib/ui/BooleanEnumDisplay.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+interface Props {
+  value: boolean;
+  options: string[];
```

Comment:
wondering if it would be more natural to replace this array by falseLabel and trueLabel that can be defaulted to false and true respectively